### PR TITLE
Note for to clear backup files

### LINKED ISSUE #46766: Add a note that after a cluster upgrade using kubeadm, backup files will need to be cleared manually.
After the upgrade, the backup directory `/etc/kubernetes/tmp` will remain. 
The ETCD backup could use several hundred MB to several GB of disk storage.
If we don't clean up the backups, the storage may become full after several upgrades.
Users may need to clean it up manually.

You can find more information about this issue here: https://github.com/kubernetes/kubernetes/issues/125357

### PR DESCRIPTION
**Pull request by tamilselvan1102**
fixes #46766

### CODE REVIEW

**Comment by tamilselvan1102** [comment]
preview page: https://deploy-preview-46773--kubernetes-io-main-staging.netlify.app/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade/#recovering-from-a-failure-state

File: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md [hunk 1]
```diff
@@ -312,6 +312,11 @@ In case of a upgrade failure and if the automatic rollback does not work, the co
 manually restored in `/etc/kubernetes/manifests`. If for some reason there is no difference between a pre-upgrade
 and post-upgrade manifest file for a certain component, a backup file for it will not be written.
 
+{{< note >}}
+   After the cluster upgrade using kubeadm, the backup directory `/etc/kubernetes/tmp` will remain and 
+   these backup files will need to be cleared manually.
```
Comment (neolit123) [inline content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md:317]:
```suggestion
After the cluster upgrade using kubeadm, the backup directory `/etc/kubernetes/tmp` will remain and 
these backup files will need to be cleared manually.
```

**Review by neolit123 — COMMENTED** [review]
SGTM, just please remove the indentation.
only docs team can merge the PR

/sig cluster-lifecycle

**Review by neolit123 — COMMENTED** [review]
/lgtm
/label tide/merge-method-squash

thanks

**Review by divya-mohan0209 — COMMENTED** [review]
Thanks for the quickfix!
/approve

**Pull request merged**